lib/command-log.js

```diff
@@ -0,0 +1,22 @@
+'use babel'
```

Comment:
Yep yep, v2.5 has more stuff in es6, and I've been slowly moving things over with [decaffeinate](http://decaffeinate-project.org/). So it'll be a little mixed while we transition but soon enough everything will be es6.